### ISSUE TITLE
how to predict the masked token ?

### ISSUE THREAD

**Issue by dixonhsiao** [google-research/electra#11]
I know that using  https://github.com/shehzaadzd/pytorch-pretrained-BERT can predict masked token for a single example, but it seems that electra is a newly released model. How exactly can I input a sentence with some masked tokens and inference with the pretrained electra model and get the predicted tokens ?

**Comment by stefan-it** [comment]
I'm not sure, how you can achieve that with the current implementation here, but there's an upcoming PR for adding ELECTRA support in Transformers library:

https://github.com/huggingface/transformers/pull/3257

so I think you can try it in a few days with Transformers 🤗

**Comment by clarkkev** [comment]
There isn't a way of doing this because ELECTRA does not predict mask tokens like BERT, it instead predicts whether tokens are "real" or "fake."
